instana/instrumentation/google/cloud/pubsub.py

```diff
@@ -0,0 +1,106 @@
+# (c) Copyright IBM Corp. 2021
+# (c) Copyright Instana Inc. 2021
+
+from __future__ import absolute_import
+
+import wrapt
+from opentracing import Format
+
+from ....log import logger
+from ....singletons import tracer
+
+try:
+    from google.cloud import pubsub_v1
+
+    logger.debug('Instrumenting Google Cloud Pub/Sub')
```

Comment:
Similar to the SQLAlchemy, we want to write this log line only after the instrumentation succeeded

Reply:
ACK

Reply:
Still misplaced 😉